### PR TITLE
remove deprecated containerPath property from DriverPreCheckInfo interface

### LINKED ISSUE #8510: Remove `containerPath` from `driver-defnitions`.
Context: #4169

### PR DESCRIPTION
**Pull request by scottn12**
Remove the `containerPath` property from the `DriverPreCheckInfo` interface in the following file: `common/lib/driver-definitions/src/urlResolver.ts`.

To replace its functionality, use `Loader.request().url`.

Issue: https://github.com/microsoft/FluidFramework/issues/8510

### CODE REVIEW

File: common/lib/driver-definitions/src/urlResolver.ts [hunk 1]
```diff
@@ -47,12 +47,6 @@ export interface IUrlResolver {
 * for driver compatibility and preload information.
 */
 export interface DriverPreCheckInfo {
-    /**
-     * @deprecated - only needed as long as long as Loader.request() does not work as intended. When
-     * Loader.request() caches and resolves pathing properly, this can be removed. #4489, #4491
-     */
-    containerPath: string;
```
Comment (anthony-murphy) [inline common/lib/driver-definitions/src/urlResolver.ts:54]: this isn't safe to remove

Comment (scottn12) [reply common/lib/driver-definitions/src/urlResolver.ts:54]: @anthony-murphy, would you be able to elaborate? I noticed that the issues mentioned in the comment are closed, but I'm still working on testing `Loader.request()` so I kept this as a draft PR for now. Do you have any additional context as to why this isn't safe to remove yet?

Comment (anthony-murphy) [reply common/lib/driver-definitions/src/urlResolver.ts:54]: it looks like people are using it, and it doesn't seem they could easily migrate. i'm also not sure why we need to deprecate it

Comment (anthony-murphy) [reply common/lib/driver-definitions/src/urlResolver.ts:54]: this is also used by our own code: https://github.com/microsoft/FluidFramework/blob/cd3c8a8f3419f0dd3f68fba4f73024dfad4a12e8/packages/drivers/odsp-driver/src/checkUrl.ts

we should make sure all usages are removed before we remove anything

Comment (scottn12) [reply common/lib/driver-definitions/src/urlResolver.ts:54]: I believe that the `containerPath` property being referenced in the file above is actually unrelated to the one I am removing, it just happens to share the same property name right?

Comment (anthony-murphy) [reply common/lib/driver-definitions/src/urlResolver.ts:54]: no. its the same. the function check url returns `import { DriverPreCheckInfo } from "@fluidframework/driver-definitions";` which is what you are changing

**Comment by scottn12** [comment]
@leeviana, would you be able to give some insight as to why you marked this property as deprecated and if you think it should still be removed? Thanks!

**Pull request closed without merging**